### ISSUE TITLE
Screenshotting part of the screen using imagemagick's import is hard

### ISSUE THREAD

**Issue by Jonta** [Airblader/unclutter-xfixes#76]
Ubuntu 21.10
i3 4.19.1-non-git
ImageMagick 6.9.11-60 Q16 x86_64 2021-01-25

Reproduce:

- Run `import test-screenshot.png` - Either from dmenu or gnome-terminal

What happens:

- Cursor disappears, even though I'm moving it
- (Drawing a rectangle, and saving a screenshot still works)

What should happen:

- Cursor should not disappear (and instead turn into the normal crosshair)

Didn't find "screenshot" in the man-pages for `unclutter` or `unclutter-xfixes`

**Comment by Airblader** [comment]
Are you sure this is related to unclutter-xfixes? As in: if you don't run it, everything works fine? Does this happen with other tools that grab the cursor, too?

**Comment by Jonta** [comment]
> Are you sure this is related to unclutter-xfixes? As in: if you don't run it, everything works fine?

Yes. `SIGTERM`-ed unclutter-xfixes now, cursor stays, and turns into a crosshair when using `import`

> Does this happen with other tools that grab the cursor, too?

Example? =)

`xbanish` is mentioned on [the ArchWiki Unclutter page](https://wiki.archlinux.org/title/Unclutter), but if that only hides the cursor during typing, it's probably not going to have an effect here

I tried with `xbanish` anyway, and the crosshair appears while running it

**Comment by Airblader** [comment]
I didn't mean replacements for unclutter, but replacements for import, eg other similar tools like maim. I'm trying to understand if the issue is specific to import or not.

**Comment by Jonta** [comment]
`maim --select test.png` shows me crosshair, both when executed from `gnome-terminal` and `dmenu`

So it seems like an `import`-thing

**Comment by Airblader** [comment]
Yeah, I would think so, too; import is quite an old tool, I have no idea how it works. I would generally recommend to use a more modern replacement like `maim` anyway, though of course you can use what you prefer. I'll leave this issue open if anyone wants to investigate this further, but I won't do so myself.

**Comment by Jonta** [comment]
`import` was just the only one I've ever used

I'll probably switch over to `maim`, alias `maim --select` to something else for less typing, and then get dmenu to read aliases, where this thread seems helpful: https://askubuntu.com/questions/1135073/dmenu-doesnt-recognise-aliases

**Comment by rdiaz02** [comment]
For the record: I was having the exact same problem with `import`, that I could trace to `unclutter`.  Switched to `maim -s` as suggested, and things are fine now.
